app/helpers/sellers_helper.rb

```diff
@@ -3,12 +3,13 @@
 module SellersHelper
   def self.generate_seller_json(seller:)
     locations = seller.locations
-    distributor = seller.distributor
+    distributor = get_deprecated_distributor(seller: seller)
```

Comment:
Also FYI @oliviasztanga @danthaman44 the distributor on the /sellers/ endpoint is being deprecated in favor of the /campaigns/ endpoint which should include distributor